src/com/vogon101/java/Renderer.scala

```diff
@@ -0,0 +1,161 @@
+package com.vogon101.java
+
+import com.colofabrix.scala.geometry.shapes.Circle
+import com.colofabrix.scala.math.Vector2D
+import com.colofabrix.scala.tankwar.Bullet
+import com.colofabrix.scala.tankwar.Tank
+import com.colofabrix.scala.tankwar.World
+import com.colofabrix.scala.tankwar.integration.TankEvaluator
+import org.lwjgl.LWJGLException
+import org.lwjgl.opengl.Display
+import org.lwjgl.opengl.DisplayMode
+import org.lwjgl.opengl.GL11
+import scala.collection.mutable.ListBuffer
+
+/**
+ * Created by Freddie on 17/05/2015.
+ */
+class Renderer (private var _world: World){
+
+  def world = _world
+  val width = world.arena width
+  val height = world.arena height
+
+  init()
+
+
+
+  def init () {
+    try {
+      initGL()
+    }
+    catch  {
+      case lwjgle : LWJGLException => {println("Could not start Graphics"); System exit 1}
+    }
+
+    setCamera()
+  }
+
+  def update() {
+    setCamera()
+    drawBG
+    drawTanks()
+    drawBullets()
+    Display.sync(25)
+    Display.update()
+
+    if (Display.isCloseRequested())
+      System.exit(0);
+  }
+
+  @throws(classOf[LWJGLException])
```

Comment:
I quite like leaving it in there so that one can see the exception thrown but it can be removed